src/Dofs/MixedDofHandler.jl

```diff
@@ -203,6 +203,9 @@ function close!(dh::MixedDofHandler{dim}) where {dim}
         push!(dh.cell_coords.length, length(cell.nodes))
     end
 
+    if return_dicts
```

Comment:
This doesn't seem to be used anywhere

Reply:
It is used for getting the `node2dof_map` in the constructor. That is needed for reordering the values according to the node order in `project`.

Reply:
Ah, I thought it was a kwarg and searched for it as that. Perhaps it would be better to have
```
function close!(dh::MixedDofHandler)
    dh, _, _, _ = _close!(dh)
end
function _close!(dh::MixedDofHandler)
    # ...
    return dh, vertexdicts, edgedicts, facedicts
end
```
and you can call `_close!` directly in the projection constructor.

Reply:
I tried to comply with how this works in the DofHandler. This looks like a nice solution, but for the MixedDofHandler there already is a `_close!` function that does something else. We could still make `return_dicts` a kwarg?

Reply:
Make it `__close!` then

Reply:
I added this for both the MixedDofHandler and the DofHandler now (in order to be consistent). I don't think it is ever used within JuAFEM for the DofHandler though (so it also shouldn't break anything).

Reply:
Nice. The reason, as I am sure you know, is that it is generally bad if output types depend on input values.